tasklist/app/controllers/tasks_controller.rb

```diff
@@ -1,6 +1,7 @@
 class TasksController < ApplicationController
 	def index
 		@tasks = Task.all
+		@tasks = @tasks.order(:complete_date, :name)
```

Comment:
These two lines could probably be combined by using `Task.all.order(:complete_date, :name)`.